### PR TITLE
New package: devpod-0.5.10

### PR DESCRIPTION
**Pull request by subado**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (X86_64-GLIB)

### CODE REVIEW

File: srcpkgs/devpod/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'devpod'
+pkgname=devpod
+version=0.5.10
+revision=1
+archs="x86_64* armv6l*"
```
Comment (abenson) [inline srcpkgs/devpod/template:5]: Why is this limited?

File: srcpkgs/devpod/template [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+makedepends="go"
+short_desc="Client-only tool to create reproducible developer environments"
+maintainer="subado <subado.main@gmail.com>"
+license="MPL-2.0"
+homepage="https://devpod.sh"
+distfiles="https://github.com/loft-sh/devpod/archive/refs/tags/v${version}.tar.gz"
+checksum=1fc1d6f273f6cc46f476a746e48acd56d5b8136846071140668c295a74f1e754
+wrksrc="${pkgname}-${version}"
```
Comment (abenson) [inline srcpkgs/devpod/template:13]: This isn't needed.

File: srcpkgs/devpod/template [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'devpod'
+pkgname=devpod
+version=0.5.10
+revision=1
+archs="x86_64* armv6l*"
+makedepends="go"
```
Comment (abenson) [inline srcpkgs/devpod/template:6]: Why can't you use the go build-style?

File: srcpkgs/go/template [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'go'
 pkgname=go
-version=1.22.0
+version=1.22.2
```
[the remark this answers is not in the document]
Comment (subado) [reply srcpkgs/go/template:3]: Yes, devpod requires at least go 1.22.2. I marked PR as "draft"

File: srcpkgs/devpod/template [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'devpod'
+pkgname=devpod
+version=0.5.10
```
Comment (abenson) [inline srcpkgs/devpod/template:3]: 0.5.12 is out

**Pull request closed without merging**